data-raw/run-test-model.R

```diff
@@ -0,0 +1,42 @@
+library(dplyr)
+devtools::load_all() # can't call library(rbabylon) from within rbabylon when using pkgr
+
+###################
+# FUNCTION DEF
+###################
+
+#' Helper to re-run models used for vignettes and testing
+#'
+#' Note: will overwrite any output directory for this model.
+#' @param .model_dir Path to directory model will be run in, relative to project/package root
+#' @param .model_name Name of model file, without extension
+run_test_model <- function(
+  .model_dir,
+  .model_name
+) {
+
+  .model_dir <- file.path(rprojroot::find_rstudio_root_file(), .model_dir)
```

Comment:
Do we need the `.` prefix?

Reply:
eh, I'm just overwriting the input variable